sklearn/compose/tests/test_column_transformer.py

```diff
@@ -111,6 +111,20 @@ def test_column_transformer():
     assert_array_equal(ct.fit(X_array).transform(X_array), X_res_both)
     assert len(ct.transformers_) == 2
 
+    # test case that ensures that the column transformer does also work when
+    # a given transformer doesn't have any columns to work on
+    ct = ColumnTransformer([('trans1', Trans(), [0, 1]),
```

Comment:
Maybe it could be a separate test? Also it might be worth checking that
```py
ColumnTransformer([('trans1', Trans(), [])])
```
is an identity operation?

Reply:
Is this what you mean?

```
ct = ColumnTransformer([('trans', Trans(), [])], 
                           remainder='passthrough')
assert_array_equal(ct.fit_transform(X_array), X_res_both)
assert_array_equal(ct.fit(X_array).transform(X_array), X_res_both)
assert len(ct.transformers_) == 2
```